cmux-tui/scripts/verify-startup-benchmark.py

```
@@ -0,0 +1,1382 @@
+            raise SystemExit(f"startup artifact manifest record is unsafe: {record!r}")
+        listed[record_path] = record
+    actual = collect_artifact_records(artifact_root)
+    if listed != actual:
+        missing = sorted(set(listed) - set(actual))
+        extra = sorted(set(actual) - set(listed))
+        changed = sorted(
+            name for name in set(listed) & set(actual) if listed[name] != actual[name]
+        )
+        raise SystemExit(
+            f"startup artifact is not closed: missing={missing}, extra={extra}, changed={changed}"
+        )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)